lib/entries.js

```diff
@@ -1,114 +1,108 @@
 'use strict';
 
 function Entries(entries) {
```

Comment:
you implemented a class, except an implicit one.

But this whole class will be removed shortly anyways. As https://github.com/stefanpenner/fs-tree-diff/blob/master/lib/index.js#L120 is more or less all (plus some bug fixes) that was needed to be implemented to make `updates` work -> https://github.com/stefanpenner/fs-tree-diff/blob/tree-to-tree/lib/index.js#L112-L114